README.md

```diff
@@ -104,6 +104,12 @@ ${DOCKER_IMAGE}
 
 ```
 
+## Versioning
+
+Semantic versioning is used and containers are tagged based on GitHub tags: If a tag,
+v1.2.3 is pushed to GitHub then a container image is built with tags `1.2.3` as well as `1.2`.
+Thus the `major.minor` tag is overwritten when the repo' is patched.
```

Comment:
not sure i understand.  do you mean the `major.minor` tag always points to the latest minor point release?

Reply:
Yes.

Reply:
ok, just wasn't real clear from docs.